synapse_data_warehouse/synapse/dynamic_tables/V2.26.2__node_latest.sql

```diff
@@ -0,0 +1,36 @@
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP
+
+CREATE DYNAMIC TABLE IF NOT EXISTS NODE_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS
+        WITH latest_unique_rows AS (
+            SELECT
+                *
+            FROM
+                {{database_name}}.synapse_raw.nodesnapshots --noqa: TMP
+            WHERE
+                SNAPSHOT_TIMESTAMP >= CURRENT_TIMESTAMP - INTERVAL '14 DAYS'
+            QUALIFY ROW_NUMBER() OVER (
+                    PARTITION BY id
+                    ORDER BY change_timestamp DESC, snapshot_timestamp DESC
+                ) = 1
+        ),
+        processed_latest_unique_rows AS (
+            SELECT
+                * EXCLUDE (annotations, reference),
+                PARSE_JSON(annotations):annotations::OBJECT AS annotations,
+                PARSE_JSON(annotations):etag::STRING AS etag,
+                PARSE_JSON(reference):targetId::STRING as reference_target_id,
+                PARSE_JSON(reference):targetVersionNumber::STRING as reference_target_version_number
```

Comment:
These last 4 lines are doing some data processing to refine `node_latest` and make it more silver 🥈 

* I'm extracting the "annotations" and "etag" keys from the JSON stored in the original `annotations` column, and putting them in their own columns. I'm also excluding the original `annotations` column from the final table.
* I'm doing the same extraction + exclusion for the `reference` column.

Reply:
I like this update, we may have to worry about the breaking change this makes. Many people are using annotations:annotations already so this would potentially be an issue

Reply:
I see - since this is a breaking change and it's December, it'd make sense to be more intentional about this: 

I can remove these changes from this PR, and pitch this type of transformation in our medallion architecture design doc (do we have a SPIKE ticket for the medallion work yet?).

In that doc, we can also discuss how we would share data structure updates to users who already have dashboards/Streamlit apps deployed. I like the pseudo release notes you've been sharing on the `synapse_datawarehouse` channel, so we could expand on that so it reaches all users.

LMK what you guys think cc @philerooski

Reply:
@jaymedina 

Just my 2cents, but I think we can create a ticket around creating our first set of dashboard-ready tables.  For example, extracting annotations _could_ be a "gold" table.

Non-backwards compatible changes would be handled like other products - I can float to our stakeholders this proposed change in the future and ask them if that is desired.  From there, we would have a timeline for implementation and migration plans